docs/building.md

```diff
@@ -184,3 +184,21 @@ build machine.
 ~~~~~~~~~~~~~{.sh}
 cmake -DRPM=ON -DCMAKE_INSTALL_PREFIX=/usr ..
 ~~~~~~~~~~~~~
+
+## Building for Peripheralmanager Android Things
+
+Change src/CMakeLists.txt:140 to the location of libandroidthings.so on your
```

Comment:
The hard-coded path to a library on brendan's machine was switched out in favor of using CMake.  Assuming the github repo below is where Android will release the definition/implementation of libandroidthings then:

Requirements: Android Things Native Library (https://github.com/androidthings/native-libandroidthings)
The Android Things Native Library contains a CMake find_package module (FindAndroidThings.cmake).  Make sure the directory containing this module is added to the CMAKE_MODULE_PATH.  Example cmake -DCMAKE_MODULE_PATH=/path/to/native-libandroidthings